src/svgcontext.ts

```diff
@@ -160,6 +168,10 @@ export class SVGContext extends RenderContext {
       ...defaultFontAttributes,
     };
 
+    this.groupAttributes = [];
+    this.applyAttributes(svg, this.attributes);
+    this.groupAttributes.push({ ...this.attributes });
+
```

Comment:
I have used SVG to place the initial attributes of SVGContext, the ones defined in the constructor.

Reply:
Looking good, but one thing to consider is whether you want to use a more React/Lit-Element style of:

```
this.groupAttributes = [...this.groupAttributes, ...this.attributes];
```

where groupAttributes is treated as immutable, and thus Proxy objects could set listeners for it to do something more complex when altered.  Just a thought for the future.

Reply:
I left it untouched to be consistent with `groups` and `state_stack`. I can make a PR to change the three if you want.